### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cpcgoogle**
Just a couple minor tweaks that I think could help a newbie.

### CODE REVIEW

File: bigquery/README.md [hunk 1]
```diff
@@ -135,6 +136,7 @@ export VM_ZONE=<the zone of the VM>
 export PROJECT=<project name>
 gcloud compute ssh $VM_NAME --project $PROJECT --zone $VM_ZONE --ssh-flag="-L" --ssh-flag="8080:localhost:8080" --ssh-flag="-L" --ssh-flag="8787:localhost:8787"
 ```
+If you are unable to connect, you may need to add a <a href="https://cloud.google.com/compute/docs/vpc/firewalls">firewall rule</a> for ports 22 and 80.
```
Comment (myounglai) [inline bigquery/README.md]: Did the port forwarding through ssh not work for some reason?  Atlas doesn't have good built in security, so it's not a good idea to open these ports in the firewall.

File: bigquery/README.md [hunk 2]
```diff
@@ -16,15 +16,15 @@
...
+cd Broadsea/bigquery
 ```
 
 ## Creating the CDM schema in BigQuery
 
-- Create two bigquery datasets named "cdm" and "ohdsi" in your cloud project at
+- <a href="https://cloud.google.com/bigquery/quickstart-web-ui#create_a_dataset">Create two bigquery datasets</a> named "cdm" and "ohdsi" in your cloud project at
   https://bigquery.cloud.google.com
-- Execute the following
-``` bash
```
Comment (myounglai) [inline bigquery/README.md]: Why did you remove the bash quoting for just this command?

File: bigquery/README.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ PostgreSQL instance in Cloud SQL:
 - Enter an instance name and default password
 - Choose a region and zone or leave the defaults
 - Click on "configuration options", select "authorize networks", and add the
-  external IP of the VM that you noted earlier
+  external IP followed by /32 of the VM that you noted earlier.
```
Comment (myounglai) [inline bigquery/README.md]: Is the /32 actually needed if you're entering a single IP?  I'm pretty sure I didn't enter it.

File: bigquery/README.md (hunk 1 above)
Comment (cpcgoogle) [reply bigquery/README.md]: Good point on the HTTP port. I may have overshot that. Maybe we just want 22, which would still needed to be open for SSH forwarding.

File: bigquery/README.md (hunk 3 above)
Comment (cpcgoogle) [reply bigquery/README.md]: I just checked it. You are correct, you don't need a to put a single IP in CIDR notation.

File: bigquery/README.md (hunk 1 above)
Comment (myounglai) [reply bigquery/README.md]: It shouldn't need to be open at all if you use "gcloud compute ssh".  Did you have problems connecting using that?
